code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -3065,13 +3066,15 @@ datum
 				if(!M) M = holder.my_atom
 				M.nutrition += 1 * mult
 
-				if(probmult(8))
-					M.emote("fart")
-
 				if(prob(3))
 					M.reagents.add_reagent("cholesterol", rand(1,2) * mult)
 				..()
-
+				
+			do_overdose(var/severity = 1, var/mob/M, var/mult = 1)
+				if (severity == 1)
+					if(probmult(16))
+						M.emote("fart")
```

Comment:
```suggestion
			do_overdose(var/severity, var/mob/M, var/mult = 1)
				if(probmult(16))
					M.emote("fart")
```
If you have more than double the overdose amount, `severity` becomes 2 - see https://github.com/goonstation/goonstation/blob/1e9e4f92c45d515c0d582e0303ff2454b26dacda/code/modules/chemistry/Chemistry-Reagents.dm#L246